sequence_processing_pipeline/Job.py

```diff
@@ -287,6 +306,10 @@ def qsub(self, script_path, qsub_parameters=None,
 
         if job_info['job_id']:
             # job was once in the queue
+            if callback:
```

Comment:
All these should be `if callback is not None`

Reply:
Ty! will do!